### ISSUE TITLE
HttpClientWithInterceptor.send

### ISSUE THREAD

**Issue by MichelFortes** [CodingAleCR/http_interceptor#37]
Hello guys. 
It looks like `httpClientWithInterceptor.send(BaseRequest request)` doesn't use interceptors, retry policies and request timeouts.
Is this a desired behavior?
Best regards

**Comment by CodingAleCR** [comment]
Hi, at the moment yes, send is used internally and I have not worked on adding support for it just yet. It is, however, in the close roadmap goals.

**Comment by CodingAleCR** [comment]
Using `send` method was added on #98 and to the 2.0.0 beta, feel free to try it out if you want ;)

**Comment by dexcell** [comment]
@CodingAleCR 
`send` method still not available on 2.0.0 beta2
Could you please add to 2.0.0 beta3

I need to post form data with multipart  using`send`
perhaps something like this:
````
// Create normal http request
http.MultipartRequest request = new http.MultipartRequest("POST", Uri.parse("$baseUrl/account/profile"));

// Add some file
request.files.add(await http.MultipartFile.fromPath(entry.key, entry.value.path, contentType: MediaType('image', 'jpeg')));

// Use intercepted http to send the request
final http.StreamedResponse response = await interceptedHttp.send(request);
````
